ROS/osr/src/roboclaw_wrapper.py

```diff
@@ -147,15 +152,22 @@ def read_encoder_values(self):
             enc_msg.effort.append(current)
 
         self.current_enc_vals = enc_msg
-
+        
     def corner_cmd_cb(self, cmd):
-        r = rospy.Rate(10)
-        rospy.logdebug("Corner command callback received: {}".format(cmd))
```

Comment:
These debug messages should stay here since that's when they've actually been received.

Reply:
Debug message should go back in, the Rate isn't needed anymore though as there is no threadlock to sleep in though i believe